source/manual/alerts/publisher-unprocessed-fact-check-emails.html.md

```diff
@@ -0,0 +1,44 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 'Publisher: Unprocessed fact-check emails'
+section: Icinga alerts
+subsection: Email alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-07-02
+review_in: 6 months
+---
+
+### Unprocessed fact-check emails
+
+This means there are unprocessed emails in the fact-check inbox
+(govuk-fact-check@digital.cabinet-office.gov.uk).
+Publisher has a [script (`mail_fetcher`)](script) that checks the inbox
+[every 5 minutes][schedule] and [processes all emails present][process].
+
+If emails are not processed or deleted, this alert is triggered. The level of
+the [alert][] depends on the number of emails present - which was chosen
+arbitrarily and is currently:
+- warning with 1 email
+- critical with 2 or more emails
```

Comment:
It might be worth linking to the alert directly here in case the numbers change in the future.